net-im/skypeforlinux/skypeforlinux-5.1.0.1.ebuild

```diff
@@ -18,36 +20,36 @@ S="${WORKDIR}"
 QA_PREBUILT=opt/skypeforlinux/skypeforlinux
 RESTRICT="mirror bindist strip" #299368
 
-RDEPEND="dev-libs/atk
-	dev-libs/expat
-	dev-libs/glib:2
-	dev-libs/nspr
-	dev-libs/nss
-	gnome-base/gconf:2
+RDEPEND="dev-libs/atk[${MULTILIB_USEDEP}]
+	dev-libs/expat[${MULTILIB_USEDEP}]
+	dev-libs/glib:2[${MULTILIB_USEDEP}]
+	dev-libs/nspr[${MULTILIB_USEDEP}]
+	dev-libs/nss[${MULTILIB_USEDEP}]
+	gnome-base/gconf:2[${MULTILIB_USEDEP}]
 	gnome-base/libgnome-keyring
```

Comment:
Oh my... Correct me if I'm wrong but this would require a lot of work. We should make dependencies of libgnome-keyring multilib compatible first (e.g. gnome-base/gnome-keyring, dev-libs/gobject-introspection) and then their deps and deps of deps, right?